example/imageSender/README.md

```diff
@@ -1,12 +1,20 @@
 # HOW TO BUILD
 1. Make sure you have CMAKE 3.0.0 or above installed
 2. Install boost/program_options library
-3. Download latest release from https://github.com/ni/adas-replay-hil-image-shim/tags
-4. Extract the content in the release to the root of this folder
-5. If you are on Linux platform, create a symbolic link to the linux shared library
+3. copy ```boost``` to ```local``` folder
+
+![image](https://github.com/ni/adas-replay-hil-simulator-sdk/assets/95634231/78121545-b04d-41b5-b0e4-4ab42da1b59c)
```

Comment:
This is specific to Windows. We should either 1) make documentation for Windows separately or 2) make universal documentation.